modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_22.04.adoc

```diff
@@ -0,0 +1,285 @@
+= Server Preparation for Ubuntu 22.04
+:toc: right
+:toclevels: 1
+:page-partial:
+:ubuntu_upgrade_url: https://linuxconfig.org/how-to-upgrade-ubuntu-to-22-04-lts-jammy-jellyfish
+:php-common: https://tecadmin.net/enable-disable-php-modules-ubuntu/
+:ondrej-php-url: https://launchpad.net/~ondrej/+archive/ubuntu/php
+:libsmbclient-php_url: https://github.com/eduardok/libsmbclient-php
+:update-alternatives_url: https://manpages.ubuntu.com/manpages/focal/en/man1/update-alternatives.1.html
+:phpmyadmin_latest_url: https://www.phpmyadmin.net/downloads/
+:pear-package_url: https://pear.php.net/package/PEAR/
+:page-aliases: installation/manual_installation/server_prep_ubuntu_18.04.adoc
```

Comment:
```suggestion
```
Removing the page alias, as we do not have a legacy here and it will cause build errors (double referencing)